modules/drivers/audio/config/respeaker.pb.txt

```diff
@@ -0,0 +1,17 @@
+speaker_model: RESPEAKER
+channel_number: 6
```

Comment:
this should be `.../conf/repeaker.pb.txt` to be consistent with your settings in dag and launch.